packages/core/services/FileService/HttpFileService/test/HttpFileService.test.ts

```diff
@@ -44,6 +44,34 @@ describe("HttpFileService", () => {
         });
     });
 
+    describe("editFile", () => {
+        const httpClient = createMockHttpClient([
+            {
+                when: () => true,
+                respondWith: {},
+            },
+        ]);
+
+        it("fails if unable to find id of annotation", async () => {
+            // Arrange
+            const httpFileService = new HttpFileService({
+                baseUrl,
+                httpClient,
+                downloadService: new FileDownloadServiceNoop(),
+            });
+
+            // Act / Assert
+            try {
+                await httpFileService.editFile("file_id", { ["Color"]: ["red"] });
+                expect(false, "Expected to throw").to.be.true;
```

Comment:
I think this is redundant?

Reply:
I'm not the author of this one so I may not have the right explanation, but I think it's a method of throwing custom error messages in chai?